EKF/ekf_helper.cpp

```diff
@@ -982,12 +982,62 @@ void Ekf::get_gyro_bias(float bias[3])
 	memcpy(bias, temp, 3 * sizeof(float));
 }
 
+// get the full covariance matrix
+matrix::SquareMatrix<float, 24> Ekf::covariances() const
```

Comment:
Can you use `_k_num_states`, or does that not work?

Reply:
@bkueng as said in https://github.com/PX4/ecl/pull/543#pullrequestreview-188416379, it doesn't work.